p2p/src/net/mod.rs

```diff
@@ -32,6 +32,11 @@ use crate::{config, message, message::Announcement};
 /// that each network service provider must implement
 #[async_trait]
 pub trait NetworkingService {
+    /// A generic networking transport.
+    ///
+    /// Can be used to initialize networking transport with authentification keys for example.
```

Comment:
authentication

authentification is not a word... I still to this day wonder why people say that :smile:

Reply:
Fixed, thanks :smile: